### PR TITLE
Update container-lifecycle-hooks.md

### PR DESCRIPTION
**Pull request by ydFu**
* sync with english version in container-lifecycle-hooks.md

Signed-off-by: ydFu <ader.ydfu@gmail.com>

### CODE REVIEW

File: content/zh/docs/concepts/containers/container-lifecycle-hooks.md [hunk 1]
```diff
@@ -103,12 +103,12 @@ Resources consumed by the command are counted against the Container.
 
 When a Container lifecycle management hook is called,
 the Kubernetes management system execute the handler according to the hook action,
-`exec` and `tcpSocket` are executed in the container, and `httpGet` is executed by the kubelet process.
+`httpGet` and `tcpSocket` are executed by the kubelet process, and `exec` is executed in the container.
 -->
```
Comment (Aut0R3V) [inline content/zh/docs/concepts/containers/container-lifecycle-hooks.md:107]: I feel like this can be approved for the English portion.

**Review by Aut0R3V — APPROVED** [review]

**Review by tanjunchen — COMMENTED** [review]
/lgtm

**Review by tanjunchen — COMMENTED** [review]
/lgtm

**Pull request merged**